CHANGELOG.md

```diff
@@ -4,11 +4,23 @@ Please note that this application has changed its name over time.
 It was first called Evopedia (and was using the file format of Evopedia).
 Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed to Kiwix-JS.
 
-+## Kiwix-JS v3.2.0
-
-Released on *TODO*
-
-Detailed changelog: TODO
+## Kiwix-JS v3.2.0
+
+Released on *2021-08-22*
+
+* NEW: Support latest format of ZIM archives (with no namespace)
+* NEW: Optimization of title search by eliminating redundancies
+* NEW: Support new format of title listings (v1) in no-namespace ZIM archives
+* NEW: 'Home' key can now be used to focus the search field
+* NEW: Use native Promises (or modern polyfill where required) instead of Q
+* NEW: Use fast binary WASM decoders with fallback to ASM if necessary
+* UPDATE: Clearer and more extensive documentation for end users in About
+* UPDATE: Code testing updated to latest QUnit and moved to GitHub + SauceLabs
```

Comment:
It would be worth having 2 different lines : 
```suggestion
* UPDATE: Unit testing updated to latest QUnit, and use Karma instead of Nightwatch
* UPDATE: Replace Travis by GitHub Actions for the continuous integration
```